presto-jdbc/src/main/java/com/facebook/presto/jdbc/ColumnInfo.java

```diff
@@ -227,6 +233,8 @@ private static int getType(TypeSignature type)
                 return Types.DATE;
             case "decimal":
                 return Types.DECIMAL;
+            case "float":
+                return Types.REAL;
```

Comment:
If you look at BNF provided in 6.1 (format) you'll see that precision for FLOAT is optional. I'd say it does not matter if we introduce REAL first and FLOAT as an alias afterwards. Or introduce non-parametrized FLOAT + add an alias for REAL later.  Parametrization of FLOAT would be an extension (if we decide we want to support it) to what we have already anyway.
So to not add additional work (since both approaches are equal) I'd leave it as is + add a REAL alias when FLOAT is merged. Later we can work on parametrized FLOAT.
Last but not least: in all discussions with @martint and @raghavsethi so far we referred to this data type as FLOAT, so I'd say it's safer to add "REAL" alias to what we all know and used to use anyway than rework it and add unparametrized FLOAT alias afterwards anyway.
Moreover, in Java we know FLOAT type, so it's more natural to refer (internally) to the approximate numeric type with arbitrary defined precision (smaller than double) as FLOAT than REAL. Though, providing an alias to comply to the standard.